src/notifier/telegram_notifier.py

```diff
@@ -0,0 +1,45 @@
+# std
+import http.client
+import logging
+import json
+from typing import List
+
+# project
+from . import Notifier, Event, EventType
+
+
+class TelegramNotifier(Notifier):
+    def __init__(self, title_prefix: str, config: dict):
+        logging.info("Initializing Telegram notifier.")
+        super().__init__(title_prefix, config)
+        try:
+            self.bot_token = config["bot_token"]
+            self.chat_id = config["chat_id"]
+        except KeyError as key:
+            logging.error(f"Invalid config.yaml. Missing key: {key}")
+
+    def send_events_to_user(self, events: List[Event]) -> bool:
+        errors = False
+        for event in events:
+            if event.type == EventType.USER:
+                request_body = json.dumps(
+                    {
+                        "chat_id": self.chat_id,
+                        "text": f"*{self._title_prefix} {event.service.name}*\n{event.message}",
+                        "parse_mode": "Markdown",
```

Comment:
We could disable the notification in case of a low event priority.
```suggestion
                        "parse_mode": "Markdown",
                        "disable_notification": event.priority == EventPriority.LOW,
```